pkgs/desktops/plasma-5/ksystemstats.nix

```
@@ -0,0 +1,11 @@
+{ mkDerivation, lib
+, extra-cmake-modules
+, libksysguard, libnl, lm_sensors, networkmanager-qt
+}:
+
+mkDerivation {
+  name = "ksystemstats";
+  NIX_CFLAGS_COMPILE = [ "-I${libksysguard.bin}/share" ];
```

Just a bit of future-proofing:

```suggestion
  NIX_CFLAGS_COMPILE = [ "-I${lib.getBin libksysguard}/share" ];
```